diff --git a/src/proto_genesis_babylonnet/lib_protocol/data.ml b/src/proto_genesis_babylonnet/lib_protocol/data.ml
index 061e8c8..f89b553 100644
--- a/src/proto_genesis_babylonnet/lib_protocol/data.ml
+++ b/src/proto_genesis_babylonnet/lib_protocol/data.ml
@@ -102,7 +102,7 @@ module Pubkey = struct
 
   let default =
     Signature.Public_key.of_b58check_exn
-      "edpkugeDwmwuwyyD3Q5enapgEYDxZLtEUFFSrvVwXASQMVEqsvTqWu"
+      "genesis_key_placeholder"
 
   let get_pubkey ctxt =
     Context.get ctxt pubkey_key >>= function
diff --git a/src/proto_genesis_carthagenet/lib_protocol/data.ml b/src/proto_genesis_carthagenet/lib_protocol/data.ml
index 3956b88..f89b553 100644
--- a/src/proto_genesis_carthagenet/lib_protocol/data.ml
+++ b/src/proto_genesis_carthagenet/lib_protocol/data.ml
@@ -102,7 +102,7 @@ module Pubkey = struct
 
   let default =
     Signature.Public_key.of_b58check_exn
-      "edpktiJknwZ8rQkkZdQUBVmG8Goau6C6JBTMX1hohxMGsQAG1qfFze"
+      "genesis_key_placeholder"
 
   let get_pubkey ctxt =
     Context.get ctxt pubkey_key >>= function
